php-apache/usr/local/share/php/common_functions.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+do_build_permissions() {
+  if [ "$IS_NFS" -ne 0 ]; then
+    setfacl -R -m "d:u:$CODE_OWNER:rwX" -m "u:$CODE_OWNER:rwX" /app/
+  else
+    chown -R "$CODE_OWNER:$CODE_GROUP" /app/
```

Comment:
I think this wont work on some volume types, where the symfony image is currently doing `chmod +rw` instead